pkg/lang/python/imports_test.go

```diff
@@ -253,14 +262,14 @@ func TestResolveFileDependencies(t *testing.T) {
 			name: "import full module and use method",
 			input: map[string]string{
 				"main.py": `
-import other
-other.hello_world()
-`,
+		import other
```

Comment:
minor, but why (a) the indentation change, and (b) indenting to a rather odd column (not full-left, but also not indented relative to the surrounding code)

Reply:
Probably happened when I commented the tests out, then un-commented. I'll revert.